port/win/env_win.cc

```diff
@@ -61,7 +62,8 @@ typedef std::unique_ptr<void, decltype(FindCloseFunc)> UniqueFindClosePtr;
 
 void WinthreadCall(const char* label, std::error_code result) {
   if (0 != result.value()) {
-    fprintf(stderr, "pthread %s: %s\n", label, strerror(result.value()));
+    fprintf(stderr, "pthread %s: %s\n", label,
+            errnoStr(result.value()).c_str());
```

Comment:
Not that it is related to this fix, but shouldn't this be WinThread, not pthread?